css-typed-om/Overview.bs

```diff
@@ -1419,28 +1419,40 @@ appear as computed values (i.e. as a value stored on computed
 Computed {{CSSUnparsedValue}} objects {#computed-unparsedvalue-objects}
 -----------------------------------------------------------------------------
 
-Custom property references are resolved as part of style computation. Accordingly,
-computed {{CSSUnparsedValue}} objects will not contain {{CSSVariableReferenceValue}} objects.
-As a result, only a single {{DOMString}} will appear in the sequence contained by
-computed {{CSSUnparsedValue}} objects.
-
-Furthermore, values that at specified value time contained custom property references
-are renormalized after computation.
+A property with a specified {{CSSUnparsedValue}} value will not
+compute to a {{CSSUnparsedValue}}. Instead, after custom property references
```

Comment:
will not automatically compute

Because they *can* still compute to that, if they're custom properties without a syntax.